Source/WorkersES6/decodeDraco.js

```diff
@@ -276,12 +276,12 @@ function decodePrimitive(parameters) {
   var dracoDecoder = new draco.Decoder();
 
   // Skip all parameter types except generic
-  var attributesToSkip = ["POSITION", "NORMAL", "COLOR", "TEX_COORD"];
-  if (parameters.dequantizeInShader) {
-    for (var i = 0; i < attributesToSkip.length; ++i) {
-      dracoDecoder.SkipAttributeTransform(draco[attributesToSkip[i]]);
-    }
-  }
+  // var attributesToSkip = ["POSITION", "NORMAL", "COLOR", "TEX_COORD"];
```

Comment:
@lilleyse Is this ok to merge as is? I remember you suggested commenting out the line in Model.js instead? I think it was -
https://github.com/CesiumGS/cesium/blob/e8f8fb970823b46318362fc5972e42e3832f647d/Source/Scene/Model.js#L638

Reply:
Yeah this is ok to merge as is.

Reply:
Actually, can keep this code and only enable `"NORMAL"`? Since it is working for normals we may as well take what we can get.